### ISSUE TITLE
[Accessibility] The ExpansionTileGroup items are not focusable

### ISSUE THREAD

**Issue by ervindobri** [congthanhng/Expansion-Tile-Group#57]
**Problem:**
Currently there is no way to detect if an ExpansionTileItem was focused.
From accessibility perspectives this would be required so we can adopt **correct focused UI state**.

```
ExpansionTileGroup(
      toggleType: toggleType,
      spaceBetweenItem: spaceBetweenItem,
      onItemChanged: onItemChanged,
      children: children.map((item) {
        return ExpansionTileItem.card(
          // lacks parameters to handle focused state
          initiallyExpanded: item.initiallyExpanded,
          title: item.title,
          backgroundColor: item.backgroundColor ?? backgroundColor,
          collapsedBackgroundColor:
              item.collapsedBackgroundColor ?? backgroundColor,
          expendedBorderColor: item.expendedBorderColor ?? expendedBorderColor,
          tilePadding: item.tilePadding ?? tilePadding,
          iconColor: item.iconColor ?? colors(context).iconBrandBase,
          collapsedIconColor:
              item.collapsedIconColor ?? colors(context).iconNeutralBase,
          children: item.children,
        );
      }).toList(),
    );
```

**Expected behaviour:**
The ExpansionTileGroup and its ExpansionTileItem children widgets should be allowing customized focus traversal and navigation by allowing modifications to its UI.

**Proposition:**
Since ExpansionTileGroup.children is required to have ExpansionTileItem type, we could add a focusNode to each children as extra parameter even or other parameters to handle focused border, focused background etc.
Custom semantics are also not possible to add but that's another issue regarding accessibility.

**Comment by congthanhng** [comment]
hi @ervindobri , thank for using this package. Did you try to use **Option 2: Using ExansionGroupController**
you can find the docs here. https://pub.dev/documentation/expansion_tile_group/latest/topics/Group%20features-topic.html
If it still not work, give me the feedback at this conversation. Thanks
